src/components/UpcomingEvents.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Link } from 'gatsby'
+
+const UpcomingEvents = () => {
+  return (
+    <section>
+      <h1>No Upcoming Events</h1>
+      <p>
+        Join our <Link to="">Meeting group</Link> to stay up to date on what's
+        coming up.
+      </p>
+      <button>
```

Comment:
No need to put this in a button. We can style the link to _look_ like a button later on.